ansible/files/telemetry.sh

```diff
@@ -168,11 +172,41 @@ jupyter serverextension enable --py jupyter_spark --user
 jupyter nbextension install --py jupyter_spark --user
 jupyter nbextension enable --py jupyter_spark --user
 
+# Create analyses dir whether we are mounting efs or not
+mkdir -p $HOME/analyses
+
+if [ -n "$EMAIL" ]; then
+    # Mount entire efs locally to check for user's dir 
+    mkdir -p /mnt/efs
+    sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 fs-d0c30f79.efs.us-west-2.amazonaws.com:/ /mnt/efs
+
+    if [ ! -d "/mnt/efs/$EMAIL" ]; then
+        # If no dir for this user, create it, set ownership, add examples
+        sudo mkdir -p "/mnt/efs/$EMAIL"
+
+        sudo chown hadoop:hadoop "/mnt/efs/$EMAIL"
```

Comment:
EFS doesn't support quotas like you're talking about [0], and I don't see anything relevant on their docs page either [1]. Initially, people won't be able to access these drives from their personal machines, so I can't really see a use case for storing large amounts of data, or any data really. Probably just notebooks.

at $0.3 per GB-Month, I would be surprised if we reached $100 a month.

[0] Question about per-user quotas: https://forums.aws.amazon.com/thread.jspa?threadID=235686
[1] Docs on limits: http://docs.aws.amazon.com/efs/latest/ug/limits.html